relay-sampling/src/lib.rs

```diff
@@ -316,6 +316,27 @@ impl NotCondition {
     }
 }
 
+/// Has condition combinator.
+///
+/// Creates a condition that is true when a specific field is present.
+#[derive(Debug, Clone, Serialize, Deserialize)]
+pub struct HasCondition {
+    field: String,
+}
```

Comment:
AFAIK This should also work with a composite `{"not": {"eq": null}}` condition. The reason why I would prefer that is that it is already supported by external relays.

Reply:
Semantically it is equivalent yes, I can for sure change that. Thanks for the heads up!